File: app/src/main/java/com/appchamp/wordchunks/ui/main/OnMainFragmentClickListener.java
package com.appchamp.wordchunks.ui.main;

/**
 * Callback that listens to the events from the main fragment.
 */
interface OnMainFragmentClickListener {

    void startGameActivity(String levelId);

    void showGameFinishedFragment();

    void showPacksActivity();

    void showSlidingMenu();
}

File: app/src/main/java/com/appchamp/wordchunks/ui/main/OnMainFragmentClickListener.kt
package com.appchamp.wordchunks.ui.main

/**
 * Callback that listens to the events from the main fragment.
 */
internal interface OnMainFragmentClickListener {

    fun startGameActivity(levelId: String?)

    fun showGameFinishedFragment()

    fun showPacksActivity()

    fun showSlidingMenu()
}
